ui/assets/index.html

```diff
@@ -8,8 +8,10 @@
     />
     <title>InfluxDB 2.0</title>
     <base href="/" />
+    <%= htmlWebpackPlugin.options.header  %>
```

Comment:
the production build of `ui` injects some google analytics code here and in the body.